go.mod

```diff
@@ -365,3 +365,6 @@ replace github.com/theupdateframework/go-tuf v0.3.1 => github.com/DataDog/go-tuf
 
 // excluded as DataDog/agent-payload/v5 v5.0.59 has been removed from source directory but is still present in proxy package
 exclude github.com/DataDog/agent-payload/v5 v5.0.59 // indirect
+
+// This replace can be removed once the dependencies have been updated to use golang.org/x/net@v0.7.0
+replace golang.org/x/net v0.5.0 => golang.org/x/net v0.7.0
```

Comment:
This shouldn't be necessary.  The require for this module can be updated here directly.  I was able to change `v0.5.0` to `v0.7.0` , run `go mod tidy` and get the expected result.

Reply:
Thanks. I will be closing this PR and updated this in this new PR: https://github.com/aws-observability/aws-otel-collector/pull/1892.